src/utils.js

```diff
@@ -34,3 +34,23 @@ export function useInterval({ callback, delay, runOnStart }) {
     }
   }, [delay, runOnStart]);
 }
+
+export function isFeatureEnabled(feature) {
```

Comment:
What do you think about moving these changes to his own PR?

Reply:
Yes I will move this out of this PR.

Reply:
Moved to https://github.com/FromDoppler/doppler-webapp/pull/382